docs/result.md

```diff
@@ -0,0 +1,136 @@
+# Result
+
+This document will describe, how certain SBOM results should be deducted.
+
+## Basics
+
+A package might have a name, a version, and dependencies.  
+This information is usually stored in a `package.json` file.
+
+## Project -> `bom.metadata.component`
+
+TBD
+
+## Package -> `...component`
+
+TBD
+
+## Bundled dependencies -> `...component.components`
+
+Some projects might have [`bundleDependencies`](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#bundledependencies),
+which means, that dependencies are part of a package
+and the dependencies of each of these bundled dependencies are part of that package, too.  
+This results in a bundle where the project itself and all the `bundleDependencies` are shipped as one blob.
+
+When rendering a CycloneDX document:
+
+* `bundleDependencies` should be rendered as subcomponents of a `component`.
+* `bundleDependencies` should be treated as regular `dependencies`.
+
+Example:
+
+```json
```

Comment:
got feedback from @stevespringett : 

> I looks like the correct approach. I think just build out the TBD docs for both the metadata component and the root components. You may want to include two different examples so that people do not get confused.